src/services/http/index.ts

```diff
@@ -46,7 +53,7 @@ class Http {
     })
       .then(this.checkStatus.bind(this))
       .then(res => this.parseJSON<T>(res))
-      .catch(this.throwError);
+      .catch(err => this.handleError(err, notifyError, customErrorMessage));
```

Comment:
Where do we have custom messages in code? I saw that you've replaced all try..catches in code, but you didn't pass any error messages. We have to add them ourselves or how does it work, could you explain please?